docs/content/python/new-roles/examples.md

```diff
@@ -53,28 +55,30 @@ roles**.
 First, we'll define how organizations and repositories are related.
 
 {{< literalInclude
-    path="examples/gitclub-sqlalchemy-flask-react/backend/app/authorization.polar"
-    from="docs: begin-repo-parent"
-    to="docs: end-repo-parent"
-    gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
-    linenos=true
->}}
+path="examples/gitclub-sqlalchemy-flask-react/backend/app/authorization.polar"
+from="docs: begin-repo-parent"
+to="docs: end-repo-parent"
+gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
+linenos=true
 
-This `parent(child, parent)` rule defines that an organization is
+> }}
```

Comment:
pls turn off the autocorrecter so it's easier to see what was actually changed 🙏

Reply:
I know I always forget but there's no way to automatically turn it off for just one language